stdlib/bytes.mli

```diff
@@ -173,6 +173,28 @@ val mapi : (int -> char -> char) -> bytes -> bytes
     index (in increasing index order) and stores the resulting bytes
     in a new sequence that is returned as the result. *)
 
+val fold_left : ('a -> char -> 'a) -> 'a -> bytes -> 'a
+(** [fold_left f x b] computes
+    [f (... (f (f x b.[0]) b.[1]) ...) b.[n-1]],
+    where [n] is the length of the bytes structure [b]. 
```

Comment:
"bytes structure" seems a bit awkward, and isn't used elsewhere in the existing documentation. Also, the rest of the documentation seems to use `s` rather than `b` for the referenced bytes object. So, I'd replace `b` with `s` and say

```
where [n] is the length of [s].
```
and leave it at that. Ditto for `fold_right` I think.

Reply:
Good point. Fixed.